deployments/jwebbinar/image/global_bashrc

```diff
@@ -0,0 +1,6 @@
+export PS1="${debian_chroot:+($debian_chroot)}\u@notebook:\w\$ "
```

Comment:
Proposing keeping this one  or  moving to to environments root.

Reply:
As I commented  above,  this file  is referred to by Dockerfile.trailer so we should probably keep it where it is or move it to environments in both Dockerfile.trailer and all the projects.